src/appleseed.python/__init__.py

```diff
@@ -29,3 +29,4 @@
 
 from _appleseedpython import *
 from logtarget import *
+from TXConverter import *
```

Comment:
we don't need to import TXConverter here. Most users of appleseed.python will not use it.